docs/photutils/psf.rst

```diff
@@ -28,10 +28,379 @@ model-fitting analysis, regardless to exactly what kind of model is actually
 being fit.  We take this road, using "PSF photometry" as shorthand for the
 general approach.
 
+PSF Photometry in Crowded Fields
+--------------------------------
+
+Photutils provides an implementation of the DAOPHOT algorithm
+(`~photutils.psf.DAOPhotPSFPhotometry`) proposed by `Stetson in his
+seminal paper <http://adsabs.harvard.edu/abs/1987PASP...99..191S >`_ for 
+crowded-field stellar photometry.
+The DAOPHOT algorithm consists in applying the loop FIND, GROUP, NSTAR,
+SUBTRACT, FIND until no more stars are detected or a given number of
+iterations is reached.
+
+Basically, DAOPHOT works as follows. The first step is to estimate the sky
```

Comment:
Maybe `DAOPHOT`-> here to `` `~photutils.psf.DAOPhotPSFPhotometry` `` so that it's clear this is referncing how the photutils version works?